synapse/handlers/appservice.py

```diff
@@ -74,7 +74,7 @@ def notify_interested_services(self, current_id):
                 limit = 100
```

Comment:
I think it would help avoid confusion if `upper_bound = self.current_max` above this line was removed